src/scripts/main.js

```diff
@@ -13,7 +32,7 @@ pushNotification(
 );
 
 pushNotification(
-  150,
+  160,
   10,
   'Title of Error message',
   'Message example.\n ' + 'Notification should contain title and description.',
```

Comment:
This comment is identical to the previous one. You should use template literals for better readability.